addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -323,9 +323,15 @@ def _channel_fold(self, state, state_count):
 
     def _rtc_join_call(self, store=None, check_rtc_session_ids=None):
         self.ensure_one()
-        check_rtc_session_ids = (check_rtc_session_ids or []) + self.rtc_session_ids.ids
+        session_domain = []
+        if self.partner_id:
+            session_domain = [("partner_id", "=", self.partner_id.id)]
+        elif self.guest_id:
+            session_domain = [("guest_id", "=", self.guest_id.id)]
+        user_sessions = self.search(session_domain).rtc_session_ids
+        check_rtc_session_ids = (check_rtc_session_ids or []) + user_sessions.ids
         self.channel_id._rtc_cancel_invitations(member_ids=self.ids)
-        self.rtc_session_ids.unlink()
+        user_sessions.unlink()
```

Comment:
To avoid the feature being confusing with multiple tabs in multiple calls, one user can only be in one call at once (similar to Discord).